--- ./FusionIcon/util.py	2011-10-16 17:41:01.255539647 +0900
+++ ./FusionIcon/util.py.new	2011-10-16 22:10:32.570293473 +0900
@@ -218,12 +218,12 @@
 
 		try:
 			context = compizconfig.Context( \
-				plugins=['decoration'], basic_metadata=True)
+				plugins=['decor'], basic_metadata=True)
 
 		except:
 			context = compizconfig.Context()
 
-		self.command = context.Plugins['decoration'].Display['command']
+		self.command = context.Plugins['decor'].Screen['command']
 
 		for decorator in installed.decorators:
 			self[decorator] = CompizDecorator(decorator, self, installed)
